frontend/src/components/University/DepartmentSection.jsx

```
@@ -0,0 +1,28 @@
+  return (
+    <div className="mb-4 bg-gray-50 p-3 rounded-xl shadow">
+      <div className="font-bold text-gray-800 mb-2">{deptName}</div>
+      <div className="space-y-1">
+        {visibleMembers.map((m, i) => (
+          <MemberCard key={i} username={m.username} email={m.email} />
+        ))}
+        {members.length > 4 && (
+          <button
+            className="text-sm text-blue-600 hover:underline mt-1"
+            onClick={() => setExpanded(!expanded)}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)